.landscaper/component-descriptor.yaml

```diff
@@ -0,0 +1 @@
+# the base component descriptor will be provided by the pipeline and modified by the .ci/component_descriptor script
```

Comment:
why do you need this file?
In the ci pipeline the base component descriptor from the gardener-ci is used